### PR TITLE
fix: avatar zoom icon issues

### LINKED ISSUE #2111: Avatar: zoom icon issues
In Fiori 3 visual core, the "Zoom icon" has the alignment rule: 
image
In our current version, we do not follow this alignment:
image

### PR DESCRIPTION
**Pull request by N1XUS**
## Related Issue
Closes #2111

## Description
Fixed:
* Position of the zoom icon for circle avatar
* Vertical centering for the icons and text inside avatar

### Before:
image

### After:
image

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
- [x]hover state of the element follow design spec
- [x] focus state of the element follow design spec
- [x] active state of the element follow design spec
- [x] selected state of the element follow design spec
- [x] selected hover state of the element follow design spec
- [x] pressed state of the element follow design spec
- [x] Responsiveness rules - the component has modifier classes for all breakpoints
- [x] Includes Compact/Cosy/Tablet design
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [x]`fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [x] Storybook documentation has been created/updated
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

File: src/avatar.scss [hunk 1]
```diff
@@ -64,7 +64,13 @@ $block: #{$fd-namespace}-avatar;
...
 
   &__icon {
-    @include fd-icon-element-base();
+    @include fd-icon-element-base() {
+      // Added in order to trully center the icon inside it's container
+      display: flex;
+      align-content: center;
+      justify-content: center;
+      align-items: center;
+    }
```
Comment (InnaAtanasova) [inline src/avatar.scss:73]: The `_mixins.scss` file contains reusable mixins such as `fd-flex-center`. 
Screen Shot 2021-04-28 at 11 41 51 AM

You can call here the mixin like this:
```
@include fd-flex-center() {
        align-content: center;
}
```

Comment (InnaAtanasova) [reply src/avatar.scss:73]: You can remove the comment on line 68

Comment (N1XUS) [reply src/avatar.scss:71]: Done. Please re-check

**Review by InnaAtanasova — APPROVED** [review]

**Comment by InnaAtanasova** [comment]
@N1XUS you need a commit containing the text [ci visual] to trigger chromatic visual regression. Without it the checks won't pass.

**Pull request merged**